docs/reference/markdown.md

```diff
@@ -0,0 +1,94 @@
+own "flavour" of Markdown. Be sure to know the Markdown features of the service
+you are using so that your Markdown renders properly.
+
+## Getting Started
+
+We recommend [markdownguide.org](https://www.markdownguide.org/) to be your first point of reference if you are learning
+Markdown for the first time. It covers topics like what Markdown is, its syntax, advanced tips, and the different
+services that support Markdown. Flavours of Markdown specific to a service build on top of these basics.
+
+## Sailbot and Markdown
+
+We write Markdown for GitHub and Material for MkDocs. The following sections
+detail how Markdown is used in these services.
+
+### GitHub
+
+We use Markdown in GitHub for technical documentation and collaboration. This includes:
+
+- `README.md` files
+- Issues
+- Pull Requests
+
+Almost all places where text is written in GitHub support Markdown. GitHub also allows you to preview
+your Markdown before you submit any comments.
+
+=== "Before Rendering"
+
+    ![image](../assets/images/markdown/issue_md_text.png)
+
+=== "After Rendering"
+
+    ![image](../assets/images/markdown/issue_md_rendered.png)
+
+The image above shows an example of a "write" and a "preview" tab for writing a comment on an issue. It might look
+different depending on where you are writing, but there usually exists a preview option!
+
+!!! note "GitHub-Flavoured Markdown"
+
+    GitHub uses its own "flavour" of Markdown. Certain features, like using HTML, are excluded for security reasons.
+    Visit the [official GitHub Markdown guide](https://docs.github.com/en/get-started/writing-on-github) for more
+    information on the available features.
+
+### Material for MkDocs
+
+We use Markdown in Material for MkDocs to create this website! Since it is written in Markdown, no frontend
+experience is required to contribute to our docs.
+
+Material for MkDocs supports powerful features purpose-built to take technical documentation to the next level.
+Feel free to browse this site to see how we use these features, exploring their syntax in the
+[source code](https://github.com/UBCSailbot/docs/tree/main/docs). Since GitHub renders Markdown files automatically
+you will need to click the "Raw" button to view their contents.
+
+!!! note "Material-Flavoured Markdown"
+
+    Material for MkDocs' flavour of Markdown extends upon vanilla Markdown, adding features such as admonitions 
+    (like this note) and content tabs. Refer to the
+    [official Material for MkDocs reference page](https://squidfunk.github.io/mkdocs-material/reference/) for more
+    information on the available features.
+
+## Rendering Markdown Locally
+
+You have a few choices to render Markdown on your computer.
+Be advised that if you are using an extended version of Markdown, you will
+need to consult the documentation from the service provider to render their flavour of Markdown properly. The following
+resources are good for rendering vanilla Markdown:
```

Comment:
I'm thinking we can this section to just "Rendering Markdown"
- Vanilla markdown options are good
- For GitHub, recommend drafting an issue or something and the GitHub preview VS Code extension
- For mkdocs, recommend running the site locally  with `mkdocs serve`